src/services/util.js

```diff
@@ -181,34 +205,49 @@ const displayWaarde = cijfer => {
   }
 }
 
+const getLegendLabels = (configCijfers) => {
+  const legendLabels = []
+
+  configCijfers
+    .filter(({ showInLegend }) => showInLegend)
+    .forEach(({ label }) => {
+      if (!legendLabels.includes(label)) {
+        legendLabels.push(label)
+      }
+    })
+
+  return legendLabels
```

Comment:
You can use a set here and Array.from() to convert it back into an array.
The set is constructed from configCijfers.map(...)